lib/tasks/manage.rake

```diff
@@ -66,6 +66,20 @@ namespace :manage do
     puts "#{subscribers.count} active subscribers moved from #{from_slug} to #{to_slug}"
   end
 
+  def find_subscriber_list_by_title(title:)
+    subscriber_lists = SubscriberList.where("title ILIKE '%#{title}%'")
```

Comment:
I know there's a very small risk here because it's a rake task, but would it be worth mitigating sql injection anyway? Would something like 

```
SubscriberList.where("title ILIKE ?", "%#{title}%")
```

work?

Reply:
Thanks @bevanloon I forgot about this risk! I've tested the change and updated in https://github.com/alphagov/email-alert-api/pull/799/commits/863c8405212ed00fea3e4ca96b230718368be423